tests/bgp/test_bgp_fact.py

```diff
@@ -8,7 +8,6 @@
 def test_bgp_facts(duthost):
     """compare the bgp facts between observed states and target state"""
 
-    npus = duthost.num_npus()
```

Comment:
not needed anymore ?

Reply:
npus variable is not used in this file, it was added for multi-asic support, but was removed by another PR.
So, will remove it for now and will be added back in future for supporting multi-asic testing.